server/models/Collective.js

```diff
@@ -1186,21 +1186,26 @@ function defineModel() {
 
   Collective.prototype.getChildren = function (query = {}) {
     return Collective.findAll({
+      order: ['createdAt', 'DESC'],
       ...query,
       where: { ...query.where, ParentCollectiveId: this.id },
     });
   };
 
   Collective.prototype.getEvents = function (query = {}) {
     return this.getChildren({
-      ...query,
+      order: [
+        ['startsAt', 'DESC'],
+        ['endsAt', 'DESC'],
+      ],
+      ...omit(query, ['where']),
       where: { ...query.where, type: types.EVENT },
     });
   };
 
   Collective.prototype.getProjects = function (query = {}) {
     return this.getChildren({
-      ...query,
+      ...omit(query, ['where']),
```

Comment:
Not really necessary since `where` is overridden just after

Reply:
I wanted to make the behavior more explicit. But I can rollback.